docs/root/configuration/listeners/listener_filters/tls_inspector.rst

```diff
@@ -15,8 +15,10 @@ from the client. This can be used to select a
 of a :ref:`FilterChainMatch <envoy_v3_api_msg_config.listener.v3.FilterChainMatch>`.
 
 * :ref:`SNI <faq_how_to_setup_sni>`
-* :ref:`v2 API reference <envoy_v3_api_field_config.listener.v3.ListenerFilter.name>`
-* This filter should be configured with the name *envoy.filters.listener.tls_inspector*.
+* :ref:`v3 API reference <envoy_v3_api_field_config.listener.v3.ListenerFilter.name>`
+* This filter may be configured with the name *envoy.filters.listener.tls_inspector* or
+  *type.googleapis.com/envoy.extensions.listeners.tls_inspector.v3.TlsInspector* as the
+  `type_url <https://github.com/protocolbuffers/protobuf/blob/9647a7c2356a9529754c07235a2877ee676c2fd0/src/google/protobuf/any.proto#L154>`_.
```

Comment:
@htuch do you have a better direct link for `type_url` definition?

Reply:
cc @lizan also for where we should link to.

cc @phlax as an example of what we need to automate in the next doc SoW.

Reply:
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Any.FIELDS.string.google.protobuf.Any.type_url